runtime/tests/shared/CacheDirPerm.cpp

```diff
@@ -195,37 +190,36 @@ removeTempDir(J9JavaVM *vm, char *dir)
 		char resultBuffer[J9SH_MAXPATH];
 		UDATA rc, handle;
 
-        j9str_printf(PORTLIB, baseFilePath, J9SH_MAXPATH, "%s", dir);
-        rc = handle = j9file_findfirst(baseFilePath, resultBuffer);
-        while ((UDATA)-1 != rc) {
-        	char nextEntry[J9SH_MAXPATH];
-        	/* skip current and parent dir */
-        	if (resultBuffer[0] == '.') {
-        		rc = j9file_findnext(handle, resultBuffer);
-        		continue;
-        	}
-        	j9str_printf(PORTLIB, nextEntry, J9SH_MAXPATH, "%s/%s", baseFilePath, resultBuffer);
-        	removeTempDir(vm, nextEntry);
-        	rc = j9file_findnext(handle, resultBuffer);
-        }
-        if (handle != (UDATA)-1) {
-        	j9file_findclose(handle);
-        }
-        rc = j9file_unlinkdir(baseFilePath);
-        if ((UDATA)-1 == rc) {
+		j9str_printf(PORTLIB, baseFilePath, J9SH_MAXPATH, "%s", dir);
```

Comment:
Should be sizeof(baseFilePath) rather than J9SH_MAXPATH.

Reply:
Fixed.